v3/routes/tracks.rb

```diff
@@ -8,6 +8,12 @@ class Tracks < Core
       get '/tracks/:id' do |id|
         pg :track, locals: { track: config.find(id) }
       end
+
+      get '/tracks/:id/:exercise_slug/readme' do |id, slug|
+        problem = config.find(id).find(slug)
+        problem.readme? or halt 404, { error: problem.error }.to_json
```

Comment:
Ok @kytrinyx I did this because in development trying to get the `README` for the track `ruby` and exercise `bin` was giving me some errors. Then I realised that wasn't a valid exercise just some executables :unamused:

I will remove all the extra method and add some tests